WorkAndFun/pom.xml

```diff
@@ -34,11 +34,17 @@
 		</dependency>
 
 		<dependency>
-			<groupId>com.h2database</groupId>
-			<artifactId>h2</artifactId>
-			<scope>runtime</scope>
```

Comment:
to co dotyczy h2 chyba można zakomentować albo wyciąć jak działa pod dockerem wszystko ?

Reply:
Ale to jest właśnie to co zostało usunięte - jest na czerwono :) Tego na produkcji nie ma